nconf/bugsnag.go

```diff
@@ -0,0 +1,31 @@
+package nconf
+
+import (
+	"github.com/bugsnag/bugsnag-go"
+	logrus_bugsnag "github.com/shopify/logrus-bugsnag"
+	"github.com/sirupsen/logrus"
+)
+
+type BugSnagConfig struct {
```

Comment:
Offtopic comment: Not sure if BugSnag-related code is a good candidate for this repository as I don't see is as a very opinionated tool.

Reply:
yeah...this was something that would have let us easily integrate with bugsnag..but it doesn't work that well....I don't think we can remove it right now (separate PR maybe)